### PR TITLE
Xcode 9 build fix + MobileCenter Build onboard

### PR DESCRIPTION
**Pull request by jaelim-ms**

### CODE REVIEW

File: Sasquatch/Sasquatch.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -128,6 +154,17 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		044B0BC01F7D8B000023D0A9 /* Embed Frameworks */ = {
```
Comment (clpolet) [inline Sasquatch/Sasquatch.xcodeproj/project.pbxproj]: Is this change intended?

Comment (jaelim-ms) [reply Sasquatch/Sasquatch.xcodeproj/project.pbxproj]: Yes, I added a new target for MC Build, each target requires this framework to build the app.

File: Config/Framework.xcconfig [hunk 2]
```diff
@@ -0,0 +1 @@
+CLANG_MODULES_AUTOLINK = NO
```
Comment (clpolet) [inline Config/Framework.xcconfig]: This should be added to the library scheme but not the framework scheme, when the framework is built it  takes the lib scheme.

Comment (jaelim-ms) [reply Config/Framework.xcconfig]: I really don't know why it did work. I made a change to make it work.

File: Sasquatch/Sasquatch.xcodeproj/project.pbxproj (hunk 1 above)
Comment (clpolet) [reply Sasquatch/Sasquatch.xcodeproj/project.pbxproj]: Right and they all stack up in the same project file.

File: MobileCenterAnalytics/MobileCenterAnalytics.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -863,7 +857,7 @@
...
..._NAME}.xcodeproj\" -configuration \"Release\" -target \"${TGT_NAME}\" clean\n\n# Building both architectures.\nxcodebuild -project \"${FMK_NAME}.xcodeproj\" -configuration \"Release\" -target \"${TGT_NAME}\" CLANG_MODULES_AUTOLINK=NO -sdk appletvos\nxcodebuild -project \"${FMK_NAME}.xcodeproj\" -configuration \"Release\" -target \"${TGT_NAME}\" CLANG_MODULES_AUTOLINK=NO -sdk appletvsimulator\n\n# Cleaning the oldest.\nif [ -d \"${INSTALL_DIR}\" ]\nthen\nrm -rf \"${INSTALL_DIR}\"\nfi\n\n# Creates and renews the final product folder.\nmkdir -p \"${INSTALL_DIR}\"\nmkdir -p \"${INSTALL_DIR}/Headers\"\nmkdir -p \"${INSTALL_DIR}/Modules\"\n\n# Copy the swift import file\ncp -f \"${SRCROOT}/${FMK_NAME}/Support/tvOS.modulemap\" \"${INSTALL_DIR}/Modules/module.modulemap\"\n\n# Copies the headers and resources files to the final product folder.\ncp -R \"${SRCROOT}/${WRK_DIR}/Release-appletvos/include/${FMK_NAME}/\" \"${INSTALL_DIR}/Headers/\"\n\n\n# Uses the Lipo Tool to merge both binary files (i386 + arm64) into one Universal final product.\nlipo -create \"${DEVICE_DIR}/lib${FMK_NAME}.a\" \"${SIMULATOR_DIR}/lib${FMK_NAME}.a\" -output \"${INSTALL_DIR}/${FMK_NAME}\"\n\nrm -r \"${WRK_DIR}\"";
```
Comment (clpolet) [inline MobileCenterAnalytics/MobileCenterAnalytics.xcodeproj/project.pbxproj]: We should add a comment here, I am sure I'll forget about it in a month ;)

Comment (jaelim-ms) [reply MobileCenterAnalytics/MobileCenterAnalytics.xcodeproj/project.pbxproj]: Oh man! We shouldn't forget about it!!!!
I'll add a comment there.
